ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -116,8 +123,8 @@ class Predictions extends React.PureComponent {
     if (gene && gene.primateAi) {
       genePredictors.primate_ai = {
         field: 'primate_ai',
-        warningThreshold: gene.primateAi.percentile25,
-        dangerThreshold: gene.primateAi.percentile75,
+        thresholds: [undefined, undefined, gene.primateAi.percentile25.toPrecision(3),
```

Comment:
why add the `toPrecision` here?

Reply:
I added it because they display oddly (having many digits) on the popup compared to other in silico scores, which are constants.